docs/distribute/create-or-select-a-distribution-profile.md

```diff
@@ -88,7 +88,20 @@ With a successful build, a new version of your application will be added to your
 
 Simply go to _Build Module_ _>_ _Build Configuration_ _>_ _Distribution_ and select a distribution profile you want your build to be sent.
 
-:::info
+:::tip
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-3167-buildoutput.png' />
+
+Users have the capability to select their Android build output as .AAB (Android App Bundle) from configuration settings within their Build profile.
+
+Upon sending the .AAB build to the designated Testing Distribution profile, either automatically or manually by uploading the file directly within the Testing Distribution profile, it will be automatically converted to .APK format when shared with a Testing Group. This ensures that the receiving tester will download the shared artifact in .APK format.
```

Comment:
```suggestion
When the `.AAB` build is sent to the designated Testing Distribution profile, either automatically or manually by uploading the file directly within the Testing Distribution profile, it will be automatically converted to `.APK` format when shared with a Testing Group. This ensures that the receiving tester downloads the shared artifact in `.APK` format.
```